src/Network/Network/Cortex/VpnConnection/NewAzureRmVpnConnectionCommand.cs

```diff
@@ -158,6 +158,11 @@ public class NewAzureRmVpnConnectionCommand : VpnConnectionBaseCmdlet
             HelpMessage = "Enable BGP for this connection")]
         public SwitchParameter EnableBgp { get; set; }
 
+        [Parameter(
```

Comment:
Can we add or modify a test to verify this functionality is workign and doesn't regress?

Reply:
Modified existing test.